doc/source/jupyterhub/customizing/extending-jupyterhub.rst

```diff
@@ -24,6 +24,7 @@ The general method to modify your Kubernetes deployment is to:
 
       helm upgrade --cleanup-on-fail \
 		$RELEASE jupyterhub/jupyterhub \
+        --namespace $NAMESPACE
         --version=0.8.2 \
         --values config.yaml
```

Comment:
I find this section to be quite messy, we have a RELEASE=jhub environment variable etc not doing much work to be helpful.

Below in this document we have `--namespace jhub`, and since the current document doesn't define NAMESPACE, i suggest to make it `--namespace jhub` or make some additional changes to make it cohesive within this document.

Thanks for being an active member of the community @plant99!

Reply:
Thanks for your review @consideRatio , I've introduced a NAMESPACE variable and used it across the doc page. I hope it's consistent now.